test-cases/tests/documentation/template.md

```diff
@@ -0,0 +1,20 @@
+<Test Case Number> - <Jira Title>
+
+* Sprint # 
+
+## Description
+
+<Summary>
+
+* Jira ref link:
+* Peer contact: 
+
+
+## Prerequisites
+
+* <What is needed to complete this procedure>
+
```

Comment:
What do you envision here? Contacting the writer for a pair testing session? Something else?